website/blog/2023-01-17-ferretdb-0-8-1-version-release.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "FerretDB new version release - 0.8.1"
+slug: ferretdb-0-8-1-version-release
+author: Alexander Fashakin
+description: "FerretDB 0.8.1 - the open-source MongoDB alternative - includes new features like version availability, `distinct` command & client TLS validation, and much more."
+image: ../static/img/blog/FerretDB_v0.8.1.jpg
+tags: [release]
+---
+
+![FerretDB version 0.8.1](../static/img/blog/FerretDB_v0.8.1.jpg)
+
+<!--truncate-->
+
+[FerretDB](https://www.ferretdb.io/) 0.8.1 is here, and it's better than ever!
+On the back of our Beta release, we've gone even further to improve FerretDB with lots of new features and enhancements.
+This FerretDB release and the continued support of our amazing community further strengthens our drive to bring you the open source alternative to MongoDB.
+
+Keep reading to learn more.
+
+## What's new
+
+We'll start off with probably the most anticipated update!
+We are delighted to announce that we now report the availability of newer versions of FerretDB in mongosh.
+However, this is only available for users with telemetry enabled.
+If telemetry is enabled and a newer version of FerretDB is available, you'll be notified and can stay up-to-date with the latest and best version of FerretDB.
+
+If you are yet to enable telemetry and you’d love to access this feature, please [see our documentation here](https://docs.ferretdb.io/telemetry/).
+
+Next, we've implemented the `distinct` command in FerretDB.
+With this command, you can easily find the unique values of specific fields in your data.
+The `distinct` command takes three arguments: `distinct`, `key`, and `query`.
+See usage below:
+
+```js
+db.collection.distinct(
+    <key>,
+    {
+        <query>
+    }
+)
+```
+
+For instance, if you want to find the unique values of the "age" field in a specific "people" collection, run the command below:
+
+```js
+db.people.distinct(
+    "age",
+    {
+        "age": {
+            $gt: 25
+        }
+    }
+)
+```
+
+The `distinct` argument essentially specifies the collection you want to query, the `key` argument specifies the field you want to find unique values for, and the `query` argument allows you to filter the results.
```

Comment:
This is how it was for `runCommand`, but now we call `distinct` command, so arguments are different. If we give an example, we probably should explain how it works.

Also, it's a bit inconsistent with the second example (where we use generic <oldField> <newField>